src/routes/appRoute.js

```diff
@@ -16,7 +17,7 @@ export const onPreResponse = {
 	 */
 	method: (request, reply) => {
 		const response = request.response;
-		if (!response.isBoom || process.env.NODE_ENV === 'production') {
+		if (!response.isBoom || config.isProd) {
```

Comment:
this should be `request.server.settings.app.isProd` - you can remove the `config` import